packages/apps/patterns/react-ui/src/components/IdentityList/IdentityListItem.tsx

```diff
@@ -22,24 +24,35 @@ export const IdentityListItem = ({
       onClick={() => onClick?.()}
       data-testid='identity-list-item'
     >
-      <Avatar
-        {...{
-          variant: 'circle',
-          size: 9,
-          fallbackValue: identity.identityKey.toHex(),
-          label: <p className='text-sm truncate'>{identity.profile?.displayName ?? identity.identityKey.truncate()}</p>,
-          ...(presence === SpaceMember.PresenceState.OFFLINE && {
-            status: 'inactive',
-            description: (
-              <p className='font-system-normal text-xs text-neutral-700 dark:text-neutral-300'>
-                {t('identity offline description')}
-              </p>
-            )
-          }),
-          ...(presence === SpaceMember.PresenceState.ONLINE && { status: 'active' }),
-          slots: { labels: { className: 'block shrink overflow-hidden' }, root: { className: 'shrink-0' } }
-        }}
-      />
+      <HaloRing
+        status={
+          presence === SpaceMember.PresenceState.ONLINE
+            ? 'active'
+            : presence === SpaceMember.PresenceState.OFFLINE
+            ? 'inactive'
+            : undefined
+        }
+      >
+        <Avatar
+          {...{
+            fallbackValue: identity.identityKey.toHex(),
+            label: (
+              <p className='text-sm truncate'>{identity.profile?.displayName ?? identity.identityKey.truncate()}</p>
+            ),
+            // ...(presence === SpaceMember.PresenceState.OFFLINE && {
+            //   status: 'inactive',
```

Comment:
Avatars already support `status` badging… is `HaloRing` a presentational variation on that? If so, maybe better for `react-ui` to define its own `Avatar` that does the `HaloRing` business and prevents the normal status badging on r-c’s `Avatar`.